frontend/src/libraries/explorer-wamp/nodes.ts

```diff
@@ -84,4 +121,21 @@ export default class NodesApi extends ExplorerApi {
       throw error;
     }
   }
+
+  async getDataForMap() {
+    const finalData: IMapData[] = [];
+    const item: IMapData = {
+      validatingNodes: [],
+      nonValidatingNodes: []
+    }; 
+    
+    let [validators, nonValidators] = await Promise.all([this.getNodesForMap("validators"), this.getNodesForMap("non-validators")]);
+
+    item.validatingNodes = validators;
+    item.nonValidatingNodes = nonValidators;
+
+    finalData.push(item);
+
+    return finalData;
+  }
```

Comment:
1. This method does not belong here. If you need it, move it to the component
2. @icerove please, review the usage of this method in autoRefreshHandler, this looks like a huge hack to me

Reply:
This is not resolved yet.

@icerove Please, comment here